bin/foreman_scap_client

```diff
@@ -46,32 +46,90 @@ ensure_scan_files () {
 }
 
 load_config () {
-  local config_file="/etc/foreman_scap_client/config"
+  local config_file="/etc/foreman_scap_client/config.yaml"
 
-  # Load config file
   if [ -f $config_file ]; then
-    # shellcheck disable=SC1090
-    source $config_file
+    eval "$(awk -v policy_id="$POLICY_ID" -F": " '
+    BEGIN { policy_found="false"; }
+    /^:server:/ {
+      print "SERVER=" "\"" $2 "\"";
+    }
+    /^:port:/ {
+      print "PORT=" "\"" $2 "\"";
+    }
+    /^:timeout:/ {
+      print "TIMEOUT=" "\"" $2 "\"";
+    }
+    /^:fetch_remote_resources:/ {
+      print "FETCH_REMOTE_RESOURCES=" "\"" $2 "\"";
+    }
+    /^:http_proxy_server:/ {
+      print "HTTP_PROXY_SERVER=" "\"" $2 "\"";
+    }
+    /^:http_proxy_port:/ {
+      print "HTTP_PROXY_PORT=" "\"" $2 "\"";
+    }
+    /^:ca_file:/ {
+      print "CA_FILE=" "\"" $2 "\"";
+    }
+    /^:host_certificate:/ {
+      print "HOST_CERTIFICATE=" "\"" $2 "\"";
+    }
+    /^:host_private_key:/ {
+      print "HOST_PRIVATE_KEY=" "\"" $2 "\"";
+    }
+    /^:ciphers:/ {
+      print "CIPHERS=" "\"" $2 "\"";
+    }
+
+    /^([0-9]+):/ {
+      sub(/:/, "");
+      current_policy_id=$1;
+      if (current_policy_id == policy_id) { policy_found="true"; }
+    }
+    /^  :profile:/ {
+      print "POLICY_" current_policy_id "_PROFILE=" "\"" $2 "\"";
+    }
+    /^  :content_path:/ {
+      print "POLICY_" current_policy_id "_CONTENT_PATH=" "\"" $2 "\"";
+    }
+    /^  :download_path:/ {
+      print "POLICY_" current_policy_id "_DOWNLOAD_PATH=" "\"" $2 "\"";
+    }
+    /^  :tailoring_path:/ {
+      print "POLICY_" current_policy_id "_TAILORING_PATH=" "\"" $2 "\"";
+    }
+    /^  :tailoring_download_path:/ {
+      print "POLICY_" current_policy_id "_TAILORING_DOWNLOAD_PATH=" "\"" $2 "\"";
+    }
+    END { print "POLICY_FOUND=" "\"" policy_found "\""; }
+    ' $config_file)"
   else
     echo "Config file could not be loaded"
     echo "$config_file does not exist"
     exit 1
   fi
 
-  # Set current policy values
-  # We expect the following variables to be set in the config file
-  # Absence of a variable will cause the script to exit
-  # For empty values, the variable must be set to an empty string
+  if [ "$POLICY_FOUND" == "false" ]; then
+    echo "Policy id $POLICY_ID not found"
+    exit 1
+  fi
+
+  # Remove brackets and spaces from ciphers list
+  # Turn ["cipher1", "cipher2"] into "cipher1,cipher2" or
+  # Turn ["cipher1:cipher2"] into "cipher1:cipher2"
```

Comment:
I don't think this config entry is stable in terms of what get's into actual curl.

I mean, the original config file suggested that it should be an array of strings to specify multiple ciphers, although based on what I found regarding curl: it accepts multiple ciphers in different formats based on what curl uses as underlying library. For OpenSSL multiple ciphers should be in OpenSSL format, e.g. `DES-CBC3-SHA` and multiple ciphers should be separated by a `:`, e.g. `AES256-SHA:DES-CBC3-SHA`. For some other libs it can be comma-separated strings with a different format. In any case, curl won't accept an array, but simply a string, which just propagates to underlying library.

I guess we stick with OpenSSL anyway, so that doesn't matter much, although, I'd love to change suggested format of `:ciphers:` entry in the config to be rather
```yaml
# optional cipher list if endpoints are hardened
# must be a string with column separated ciphers in OpenSSL format, e.g.:
:ciphers: "AES256-SHA:AES128-SHA:DES-CBC3-SHA"
```
instead of the current one:
```yaml
# optional cipher list if endpoints are hardened
:ciphers: ["AES256-SHA:AES128-SHA:DES-CBC3-SHA"]
```